apiary.apib

```diff
@@ -3571,6 +3573,16 @@ Get a list of tracked time.
             + started_before: `2017-04-26T10:01:49+00:00` (string, optional)
             + ended_after: `2017-04-26T10:01:49+00:00` (string, optional)
             + ended_before: `2017-05-26T10:01:49+00:00` (string, optional)
+            + subject (object, optional)
+                + type (enum, required)
+                    + Members
+                        + company
+                        + contact
+                        + event
+                        + todo
```

Comment:
Q: shouldn't this be `task` ? I thought we were getting/trying to get rid of `todo`

Reply:
@r1dotl 👆

Reply:
The thing is, on the database is it stored as todo, so in order to use task here instead of todo we would have to convert it somewhere in the code, from task to todo, because the repository in order to work, but search for a todo.

Reply:
> The thing is, on the database it is stored as `todo`, so in order to use `task` here instead of `todo` we would have to convert it somewhere in the code, from `task` to `todo`, because the `subject_type` (db column on crm_timetracking) only allows `todo` and not `task`.

Ok, but that's just an implementation detail, right? 
(AFAICT it should need an update in `TimeTrackingSearchFiltersFactory` or `SubjectType` VO,  as well as the request validator, plus some more tests).

Reply:
Yep thats true. Let me discuss it with @marcwestermann

Reply:
Any update on this?

Reply:
My guess is that this should become `task`, since we're going to have `tasks.list` and `events.list` endpoints. Right?

Reply:
@joaoinacio @JordyDeGroeve @jenssegers yup, we have a ticket for the next sprint dedicated to this subject :)